src/api/routes/transcripts.ts

```diff
@@ -52,8 +54,37 @@ const list = procedure
   return g.transcripts;
 });
 
+const getNameMap = procedure
+  .use(authUser())
+  .input(z.string())
+  .query(async ({ input: transcriptId }) => {
+    const handlebars = await extractHandlebars(transcriptId);
+    return await createNameMap(handlebars);
+  });
+
+const updateNameMap = procedure
+  .use(authUser())
+  .input(z.record(z.string()))
+  .mutation(async ({ input: nameMap }) => {
+    await sequelize.transaction(async (transaction) => {
```

Comment:
I don't think you need a transaction here because there is only one sql statement.